scripts/views/eventView.js

```diff
@@ -9,10 +9,30 @@
   eventHandler.lookup = function (e) {
     e.preventDefault();
     var zip = $('#look-up input').val();
-    if (zip) {
-      TownHall.lookupZip($('#look-up input').val());
-      eventHandler.resetFilters();
+    if (zip && zip.length === 5) {
+      repZip = zip;
+    } else if (zip && zip.length > 5) {
+      zip = zip.match(/\d{5}/g)[0];
+      plusFour = zip.match(/[+-\s]\d{4}/g);
```

Comment:
yeah, I'm trying to handle all these different versions the user may enter, but i then format it with a dash lower.